MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/account/LandingActivity.java

```diff
@@ -4,70 +4,87 @@
 import android.net.Uri;
 import android.os.Bundle;
 import android.preference.PreferenceManager;
+import android.support.annotation.NonNull;
 import android.support.customtabs.CustomTabsIntent;
 import android.support.v4.content.ContextCompat;
 import android.support.v7.app.AppCompatActivity;
 import android.util.Log;
 import android.view.View;
 import android.widget.Button;
-import android.widget.Toast;
 
+import com.afollestad.materialdialogs.DialogAction;
+import com.afollestad.materialdialogs.MaterialDialog;
+import com.github.javiersantos.materialstyleddialogs.MaterialStyledDialog;
 import com.mapbox.mapboxandroiddemo.MainActivity;
 import com.mapbox.mapboxandroiddemo.R;
+import com.mapbox.mapboxandroiddemo.analytics.AnalyticsTracker;
 
 
 public class LandingActivity extends AppCompatActivity {
 
-  private static final String SIGN_IN_AUTH_URL = "https://api.mapbox.com/oauth/authorize?response_type=code&client_id=%s&redirect_uri=%s";
+  private static final String SIGN_IN_AUTH_URL = "https://api.mapbox.com/oauth/authorize?"
+    + "response_type=code&client_id=%s&redirect_uri=%s";
 
-  private static final String CREATE_ACCOUNT_AUTH_URL = "https://www.mapbox.com/signup/?route-to=https%3A%2F%2Fwww.mapbox"
-    + ".com%2Fauthorize%2F%3Fclient_id%3D7bb34a0cf68455d33ec0d994af2330a3f60ee636%26redirect_uri%3Dmapbox-android-dev-"
+  private static final String CREATE_ACCOUNT_AUTH_URL = "https://www.mapbox.com"
+    + "/signup/?route-to=https%3A%2F%2Fwww.mapbox"
+    + ".com%2Fauthorize%2F%3Fclient_id%3D7bb34a0cf68455d33ec0d994af2330a3f60ee636%26"
+    + "redirect_uri%3Dmapbox-android-dev-"
     + "preview%3A%2F%2Fauthorize%26response_type%3Dcode";
   private static final String CLIENT_ID = "7bb34a0cf68455d33ec0d994af2330a3f60ee636";
   private static final String REDIRECT_URI = "mapbox-android-dev-preview://authorize";
+  private boolean loggedIn;
 
   @Override
   protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
-    if (!PreferenceManager.getDefaultSharedPreferences(getApplicationContext()).getBoolean("TOKEN_SAVED", false)) {
+    loggedIn = PreferenceManager.getDefaultSharedPreferences(
+      getApplicationContext())
+      .getBoolean("TOKEN_SAVED", false);
+
+    if (!loggedIn) {
       setContentView(R.layout.activity_landing);
       getSupportActionBar().hide();
       setUpSkipDialog();
       setUpButtons();
+      AnalyticsTracker.getInstance(getApplicationContext()).viewedScreen("Create/login screen",
+        loggedIn);
     } else {
       Intent intent = new Intent(this, MainActivity.class);
       startActivity(intent);
     }
+
   }
 
   @Override
   protected void onResume() {
     super.onResume();
-
-    if (getIntent() != null && getIntent().getAction().equals(Intent.ACTION_VIEW)) {
+    if (getIntent().getBooleanExtra("FROM_LOG_OUT_BUTTON", false)) {
+      Log.d("LandingActivity", "onResume: FROM_LOG_OUT_BUTTON == false");
```

Comment:
weird, that log and `FROM_LOG_OUT_BUTTON` logic check shouldn't be part of the PR. Don't know why its deletion wasn't part of a past commit. Anyways, it's gone.